apps/sports-helsinki/src/domain/venue/VenuePageContainer.tsx

```diff
@@ -81,13 +87,14 @@ const VenuePageContainer: React.FC<VenuePageContainerProps> = ({
                   onEventsLoaded={handleSimilarEventsLoaded}
                 />
               )}
-              {/* Hide similar event on SSR to make initial load faster */}
-              {/* {isClient && showSimilarVenues && (
+              {/* Hide similar venues on SSR to make initial load faster */}
+              {isClient && showSimilarVenues && (
```

Comment:
Maybe we could start using dynamic imports in places like these, then there would be no need to use `isClient`? [dynamic imports](https://nextjs.org/docs/advanced-features/dynamic-import#with-no-ssr)